librz/analysis/p/analysis_bf.c

```diff
@@ -36,6 +36,8 @@ static void bf_syscall_read(RzILVM *vm, RzILOpEffect *op) {
 	rz_bv_free(bv);
 }
 
+static PrintfCallback cb_printf = NULL;
```

Comment:
tbh, i really do not like this...

Reply:
okay dded0c2f7faa420e246f1a5a16af8c004e969099 works too and is simpler (and has flushing)